frontend/package.json

```diff
@@ -22,7 +22,8 @@
     "vue-axios": "^2.1.5",
     "vue-i18n": "^8.17.3",
     "vue-router": "^3.2.0",
-    "vuex": "^3.4.0"
+    "vuex": "^3.4.0",
+    "vuex-persist": "^3.1.3"
```

Comment:
Bitte rückgängig machen, wird in einem anderem Branch gefixt. Sonst gibt es Merge-Konflikte.

Reply:
Außerdem package-lock changes komplett rückgängig machen